code/game/objects/items/devices/body_snatcher_vr.dm

```diff
@@ -0,0 +1,70 @@
+//Body snatcher. Based off the sleevemate, but instead of storing a mind it lets you swap your mind with someone. Extremely illegal and being caught with one s
+/obj/item/device/bodysnatcher
+	name = "\improper Body Snatcher Device"
+	desc = "An extremely illegal tool that allows the user to swap minds with the selected humanoid victim. The LED panel on the side states 'Place both heads on the device, pull trigger, then wait for the transfer to complete.'"
+	icon = 'icons/obj/device_alt.dmi'
+	icon_state = "sleevemate" //Give this a fancier sprite later.
+	item_state = "healthanalyzer"
+	flags = CONDUCT
+	slot_flags = SLOT_BELT
+	w_class = ITEMSIZE_SMALL
+	matter = list(DEFAULT_WALL_MATERIAL = 200)
+	origin_tech = list(TECH_MAGNET = 2, TECH_BIO = 2, TECH_ILLEGAL = 1)
+
+/obj/item/device/bodysnatcher/New()
+	..()
+	flags |= NOBLUDGEON //So borgs don't spark.
```

Comment:
FYI, no big deal, but this is silly. Don't need to change it, just for reference for next time. You already set flags as a variable above, so it's not like it matters what the parent 'device' has. The only reason you'd need to do this is if you were concerned with adding one and not messing with the existing ones, but considering you wipe out all the existing ones by setting flags = CONDUCT, this is kinda irrelevant. I assume these aren't going to spawn every 0.001 seconds and call New() a whole lot though so the overhead of having an additional proc call would be minimal.